content/en/docs/tasks/security/citadel-config/plugin-ca-cert/index.md

```diff
@@ -48,11 +50,11 @@ The following steps enable plugging in the certificates and key into Citadel:
         --from-file=samples/certs/cert-chain.pem
     {{< /text >}}
 
-1.  Redeploy Citadel with `global.mtls.enabled` set to `true` and `security.selfSigned` to `false`.
+1.  Redeploy Citadel with `security.selfSigned` to `false`.
```

Comment:
Is this correct for 1.5? How is this working with Istiod?

Reply:
https://github.com/istio/istio.io/pull/6644/files#diff-8 taking care of this, so I only remove global.mtls.enabled in this one for consistency.